lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -307,29 +354,46 @@ static LogicalResult verifyComponentOp(ComponentOp op) {
                               "`clk`, `reset`, and output port `done`";
 }
 
+/// Returns a new vector containing the concatenation of vectors @p a and @p b.
+template <typename T>
+SmallVector<T> concat(const SmallVectorImpl<T> &a,
```

Comment:
If this is just an internal helper, I think you can (and should?) mark it `static`.